src/Components/Components/src/RenderTree/RenderTreeDiffBuilder.cs

```diff
@@ -25,14 +28,17 @@ enum DiffAction { Match, Insert, Delete }
             ArrayRange<RenderTreeFrame> oldTree,
             ArrayRange<RenderTreeFrame> newTree)
         {
+            ComponentsProfiling.Instance.Start();
```

Comment:
Why not use `ProfilingStart`/`ProfilingEnd` here?

Reply:
Because I wanted to include this method in the coarse-grained timings.

Reply:
Aha! OK -- I see how this call is different now.